### PR TITLE
[FEATURE] Add wonolog support.

### PR DESCRIPTION
**Pull request by jenkoian**
Adds support for wonolog with the additional extra features:

* Configuration support across environments
* QueryMonitor handler added and set as default for handler for all
  environments except production and local
* Local will use null as the default handler which forces the default
  wonolog handler which is a file based stream. Logs will be written by
default to wp-content/wonolog
* Production set to use the new relic handler by default.

### CODE REVIEW

**Comment by jenkoian** [comment]
Replace with https://github.com/inpsyde/Wonolog/pull/65 should it get merged and released.

File: docs/dev/logging.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+the library it extends upon, [Monolog](https://github.com/Seldaek/monolog).
+
+By default Wonolog will use a file based streaming handler, which will output the log messages to a file. We use this when working locally and by default the logs
+are written to `wp-content/wonolog`.
+
+In non-local environments (except production) we default to a custom Query Monitor handler which will log to [Query Monitor](https://querymonitor.com/) using its [logging capabilities](https://querymonitor.com/docs/logging-variables/).
+
+In production we default to a New Relic handler which will log to New Relic using its [logging capabilities](https://docs.newrelic.com/docs/agents/php-agent/logging/log-custom-events).
+
+All of these defaults can be changed though through configuration. For example, if you wanted to change your local environment to log to Query Monitor you can update
```
[the remark this answers is not in the document]
Comment (jenkoian) [reply docs/dev/logging.md:33]: `though through` (slightly different) but can see it being confusing so will remove the `though`

[the remark this answers is not in the document]
Comment (andrewjt71) [reply docs/dev/logging.md:33]: Ah yeah my bad. Agreed better with `though` removed :)

**Review by andrewjt71 — APPROVED** [review]
Nice!

**Review by jonny-bull — APPROVED** [review]

**Pull request merged**